test/token/ERC721/ERC721Holder.behavior.js

```diff
@@ -0,0 +1,25 @@
+const ERC721Holder = artifacts.require('ERC721Holder.sol');
+
+require('chai')
+  .should();
+
+function shouldBehaveLikeERC721Holder (accounts) {
+  const tokenId = 1;
+  const creator = accounts[0];
```

Comment:
:scream: 

Consider destructuring the `accounts` array (e.g. change the function declaration to  `function shouldBehaveLikeERC721Holder ([creator])`), see #1137 for reference :sweat_smile: